VLogProcessor/src/edu/umich/robustopt/experiments/CliffGuard.java

```diff
@@ -817,7 +817,7 @@ public static void main(String[] args) throws Exception {
 
 		DatabaseLoginConfiguration deployerConfig = DatabaseLoginConfiguration.findDBAliasInList(deployer_db_alias, allDatabaseConfigurations);
 		DatabaseLoginConfiguration designerConfig = DatabaseLoginConfiguration.findDBAliasInList(designer_db_alias, allDatabaseConfigurations);
-		if (deployerConfig==null || designerConfig==null || deployerConfig.getDBname()!=designerConfig.getDBname())
+		if (deployerConfig==null || designerConfig==null || !deployerConfig.getDBname().equals(designerConfig.getDBname()))
```

Comment:
When we compare the contents of two strings, we want to use `.equals` instead of `!=`